src/models/analytics-panel.ts

```diff
@@ -81,6 +82,7 @@ interface AnalyticsPanel {
   layout: AnalyticsPanelLayout;
   cards: {
     title: string;
+    subTitle: string;
```

Comment:
As it's optional, it should be `subtitle?: string`

Reply:
Fixed